src/elements/Violin.ts

```diff
@@ -70,32 +70,36 @@ export class Violin extends StatsBase<IViolinElementProps, IViolinElementOptions
     } else {
       maxEstimate = props.maxEstimate;
     }
+
+    ctx.beginPath();
     if (this.isVertical()) {
       const { x, width } = props;
       const factor = width / 2 / maxEstimate;
-      ctx.moveTo(x, props.min);
```

Comment:
hm. usually I need to move the path first to a starting point before drawing lines. or from where else would the first line start?

Reply:
beginPath starts a sub path. Therefore you don’t need to move the cursor first before calling lineTo. If we were drawing more than one violin between beginPath and closePath you would need to move the cursor. 

The line starts at x-c.estimate * factor (bottom right) and works it way clockwise until x+c.estimate * factor at which point we do not need to draw a line back to the start since closePath will do that for us.